examples/platforms/nrf52840/README.md

```diff
@@ -1,27 +1,38 @@
 # OpenThread on nRF52840 Example
 
-This directory contains example platform drivers for [Nordic Semiconductor nRF52840 SoC][nRF52840].
+<a href="http://threadgroup.org/technology/ourtechnology#certifiedproducts">
```

Comment:
Do not start a page with an image. Rather, move it to after line 7.

Reply:
Hmm, it looks much better here :) On the preview you can see that thi image is right aligned.

Reply:
Ok, it looks fine.